src/ci/docker/run.sh

```diff
@@ -307,7 +307,14 @@ if [ "$ENABLE_GCC_CODEGEN" = "1" ]; then
   extra_env="$extra_env --env ENABLE_GCC_CODEGEN=1"
   # Fix rustc_codegen_gcc lto issues.
   extra_env="$extra_env --env GCC_EXEC_PREFIX=/usr/lib/gcc/"
-  echo "Setting extra environment values for docker: $extra_env"
+fi
+
+if [ "$DOWNLOAD_RUSTC" = "1" ]; then
+  extra_env="$extra_env --env DOWNLOAD_RUSTC=1"
```

Comment:
This doesn't seem right to me. We should just be able to pass `--env DOWNLOAD_RUSTC` below, right? That way the environment variable propagates across the docker boundary if set (or doesn't if not).